Jenkinsfile_parameterized

```diff
@@ -9,8 +9,7 @@ properties([
 ])
 
 List<Map<String, Object>> secrets = [
-  secret('address-lookup-api-key', 'ADDRESS_LOOKUP_TOKEN'),
-  secret('notify-api-key', 'NOTIFY_API_KEY')
+  secret('address-lookup-api-key', 'ADDRESS_LOOKUP_TOKEN')
```

Comment:
This is deprecated way of defining secrets, but this is used only in sandbox jenkins.

Reply:
Agree, whole file is trash as we do not use sandbox env. I might remove it in next PR.